### ISSUE TITLE
Snap grid hard to see depending on background

### ISSUE THREAD

**Issue by KartikChugh** [geegaz/GMTK_Jam_2020#1]
![image](https://user-images.githubusercontent.com/6678748/90168781-a05e5f80-dd6b-11ea-9e77-f6e2b3126f7f.png)

All those dark purple areas are valid places to put a block but either no grid appears or its very faint

**Comment by geegaz** [comment]
Yep, saw this real soon. I messed up the z-order of the grid layer ^^

Thanks for the report !

**Comment by geegaz** [comment]
Fixed in next update

**Comment by geegaz** [comment]
Well now I broke it again.
Reopening this until I add the grid back.
